### PR TITLE
fix:separate aligner from signer

### PR DESCRIPTION
**Pull request by Canny1913**
terrible fix but works

### CODE REVIEW

File: src/main/kotlin/app/revanced/cli/command/MainCommand.kt [hunk 1]
```diff
@@ -158,11 +159,15 @@ internal object MainCommand : Runnable {
         val adb: Adb? = _args.deploy?.let {
             Adb(outputFile, patcher.data.packageMetadata.packageName, _args.deploy!!, !args.mount)
         }
-        val patchedFile = if (args.mount) outputFile
-        else File(args.cacheDirectory).resolve("${outputFile.nameWithoutExtension}_raw.apk")
+        val patchedFile = File(args.cacheDirectory).resolve("${outputFile.nameWithoutExtension}_raw.apk")
 
         Patcher.start(patcher, patchedFile)
 
+        // align the inputFile and write to alignedOutput
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/cli/command/MainCommand.kt:166]: Please create a class for this too the same way the signer has.

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/revanced/cli/command/MainCommand.kt (hunk 1 above)
Comment (Canny1913) [reply src/main/kotlin/app/revanced/cli/command/MainCommand.kt:166]: Sure.

Comment (Canny1913) [reply src/main/kotlin/app/revanced/cli/command/MainCommand.kt:166]: ZipAligner already has a class so i'll modify that.

Comment (Canny1913) [reply src/main/kotlin/app/revanced/cli/command/MainCommand.kt]: Done.

**Comment by oSumAtrIX** [comment]
Your PR changes all lines and will cause merge conflicts for other PRs, please only commit the lines relevant to this PR

**Comment by Canny1913** [comment]
screw you intellij

File: src/main/kotlin/app/revanced/cli/aligning/ZipAligning.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package app.revanced.cli.aligning
+
+import app.revanced.cli.command.MainCommand
+import app.revanced.cli.command.MainCommand.logger
+import app.revanced.utils.signing.align.ZipAligner
+import java.io.File
+
+object ZipAligning {
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/cli/aligning/ZipAligning.kt:8]: Call this "Aligning" the same way "Signing" is called, then we can merge 👍

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: src/main/kotlin/app/revanced/cli/aligning/ZipAligning.kt (hunk 2 above)
Comment (Canny1913) [reply src/main/kotlin/app/revanced/cli/aligning/ZipAligning.kt]: Done.

**Pull request closed without merging**